src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,22 @@
-const humanCatDogYears = (number) => {};
+const humanCatDogYears = (number) => {
+	const human = number;
+	let cat = 0;
+	let dog = 0;
+	const oldAge = number - 2;
+	if (number === 0) {
+		cat = 0;
+		dog = 0;
```

Comment:
These values are already set on lines 3 & 4, so theres no point in reassigning them here. Instead you could just return the values directly here:
```
    return [0, 0, 0];
```